config/secrets.yml

```diff
@@ -58,10 +58,29 @@ test:
       url: <%= ENV["OPENSTAX_ACCOUNTS_URL"] || 'http://localhost:2999' %>
       stub: <%= ENV.fetch("OPENSTAX_ACCOUNTS_STUB", true) %>
 
-# Do not keep production secrets in the unencrypted secrets file.
-# Instead, either read values from the environment.
-# Or, use `bin/rails secrets:setup` to configure encrypted secrets
-# and move the `production:` environment over there.
+# Deployment template for production secrets
 
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: random(hex,80)
+  base_url: "https://{{ domain }}"
```

Comment:
~~?? won't this cause all old cookies to be invalidated every time the app starts up?~~

I had commented ^^^, but I see what we're doing now in the script below.  Maybe we should add a comment indicating what's going on so that others don't have the same reaction.

Reply:
Yep, I'll make a comment.